src/components/ui/widget.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactNode } from 'react';
+
+interface CardContent {
+  children: ReactNode;
```

Comment:
Here's a good example of giving meaning to an interface name but could be better and since this is a Props, you could add "Props" to the name


```suggestion
interface WidgetProps {
```